### DISCUSSION TITLE
I only see top of plugin UI (scaler is disabled, Win & Macbook combination)

### DISCUSSION THREAD

**Discussion by Oemfoefoe-1984** [apohl79/audiogridder#628] [Help]
hi guys,

first of all, i wanted to thank the dev of Audiogridder. This app is truly a life safer! Hopefully this 16" macbook pro is the last laptop i will ever need LOL

So im working with Cubase Pro 11 on Catalina. My server is on Win 10 PC from Scan.
Everything is connected with TB/Ethernet directly between computers.
In Audiogridder 1.1 everything works good.
In Audiogridder 1.3 i only see the top 20% of my plugin's UI. The rest is a big black screen. (see picture).

![Screen Shot 2021-06-30 at 18 33 35](https://user-images.githubusercontent.com/86776502/124107550-03cdaa80-da6e-11eb-8e2d-fc4aba40526f.png)

Im not sure if i really need 1.3 :)

Still, would be nice if somebody knows what is going on.

Cheerz

**Comment by CaBuffalo** [comment]
Hi,

By default, AG is capturing the top left of the server screen where the
plugin is located. It looks like the plugin on the server is moved down.
Can you go to your server and take a look where your plugin is?

On Thu, Jul 1, 2021 at 3:13 AM Oemfoefoe-1984 ***@***.***>
wrote:

> hi guys,
>
> first of all, i wanted to thank the dev of Audiogridder. This app is truly
> a life safer! Hopefully this 16" macbook pro is the last laptop i will ever
> need LOL
>
> So im working with Cubase Pro 11 on Catalina. My server is on Win 10 PC
> from Scan.
> Everything is connected with TB/Ethernet directly between computers.
> In Audiogridder 1.1 everything works good.
> In Audiogridder 1.3 i only see the top 20% of my plugin's UI. The rest is
> a big black screen. (see picture).
>
> [image: Screen Shot 2021-06-30 at 18 33 35]
> <https://user-images.githubusercontent.com/86776502/124107550-03cdaa80-da6e-11eb-8e2d-fc4aba40526f.png>
>
> Im not sure if i really need 1.3 :)
>
> Still, would be nice if somebody knows what is going on.
>
> Cheerz
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/628>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BR4YJKSFCVC6BD3T3TVQ5UJANCNFSM47UGCI3A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by CaBuffalo** [comment]
Sorry, please ignore my previous email. I overlooked your screenshot.

You are the first person I see about this problem. I don't have this
problem in Logic.

Did you try reloading your DAW?

On Thu, Jul 1, 2021 at 9:50 AM Kevin Ma ***@***.***> wrote:

> Hi,
>
> By default, AG is capturing the top left of the server screen where the
> plugin is located. It looks like the plugin on the server is moved down.
> Can you go to your server and take a look where your plugin is?
>
> On Thu, Jul 1, 2021 at 3:13 AM Oemfoefoe-1984 ***@***.***>
> wrote:
>
>> hi guys,
>>
>> first of all, i wanted to thank the dev of Audiogridder. This app is
>> truly a life safer! Hopefully this 16" macbook pro is the last laptop i
>> will ever need LOL
>>
>> So im working with Cubase Pro 11 on Catalina. My server is on Win 10 PC
>> from Scan.
>> Everything is connected with TB/Ethernet directly between computers.
>> In Audiogridder 1.1 everything works good.
>> In Audiogridder 1.3 i only see the top 20% of my plugin's UI. The rest is
>> a big black screen. (see picture).
>>
>> [image: Screen Shot 2021-06-30 at 18 33 35]
>> <https://user-images.githubusercontent.com/86776502/124107550-03cdaa80-da6e-11eb-8e2d-fc4aba40526f.png>
>>
>> Im not sure if i really need 1.3 :)
>>
>> Still, would be nice if somebody knows what is going on.
>>
>> Cheerz
>>
>> —
>> You are receiving this because you are subscribed to this thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/apohl79/audiogridder/discussions/628>, or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ARMAR6BR4YJKSFCVC6BD3T3TVQ5UJANCNFSM47UGCI3A>
>> .
>>
>
>
> --
> Kevin Ma
> Supported by Gmail.
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Oemfoefoe-1984** [reply]
yeah, i tried. This problem doesnt occur in 1.1, just the latest Beta.
I also tried all the different video options, it works without the UI though. I can control from the server (i have extra monitors) but its not optimal.


Thanks for your time

**Reply by Oemfoefoe-1984** [reply]
FIXED!!!

I got a new 4K LG TV (Nanocell 55") and the new Beta works fantastic!!!

Im using it with a TB3 cable to HDMI from Verbatim (€15) without a hub.

The resolution is natively scaled in Catalina "around 2560x1440" but full 4K also runs fine.


JIPPIEEEE!
